app/assets/javascripts/reports/new.js

```diff
@@ -1251,6 +1251,8 @@ function reportHandsonTableConverter() {
       });
   }
 
+  $('#reportWizardEditWarning').modal('show');
```

Comment:
What if not in edit mode?

Reply:
Modal exist only in edit mode. In new this modal not exist.